scipy/stats/morestats.py

```diff
@@ -3478,3 +3478,76 @@ def circstd(samples, high=2*pi, low=0, axis=None, nan_policy='propagate'):
         R = np.minimum(1, hypot(sin_mean, cos_mean))
 
     return ((high - low)/2.0/pi) * sqrt(-2*log(R))
+
+
+def quartile_coeff_dispersion(a, q=(0.25, 0.75), axis=0,
+                              interpolation='lower'):
+    """
+    Compute the quartile coefficient of dispersion for a given sample array.
+
+    Given two quartiles Q_low, Q_high from array samples such that
```

Comment:
For the record, I think the rendering of "Q_low" and "Q_high" as regular text are fine because they are neither code nor parameters. Personally, when reading documentation online, I'd prefer to see them as mathematical symbols (e.g. `:math: Q_{\text{low}}`), but please do not change to that because it would be difficult to read for those who use documentation in the console / IDE. I suppose they could be rendered as code, but I don't think that is necessary.